torch/_inductor/constant_folding.py

```diff
@@ -159,6 +159,10 @@ def set_env(arg):
         ):
             return self.unknown_value
 
+        # This value is at least 1 GB
+        if node.meta["val"].numel() > 2**30:
```

Comment:
Bailing out here if the node is too large (cc: @eellison). Otherwise I OOM here.